NEWS

```diff
@@ -11,6 +11,211 @@
+ * Enabled DDP support on mlx5dv
+ * Added support of cuda-managed memory on IB when ODP is available
+ * Enabled use of implicit ODP with relaxed ordering
+ * Improved GPUDirect detection support on IB
+ * Increased DC initiator default count to 32 for performance optimization
+ * Added ConnectX-8 device support to IB
+ * Improved logs by suggesting to increase PF_LOG_BAR_SIZE when UAR allocation fails
+ * Added support for subnet filter list on RoCE GID
+ * Added a reason when a resource's lane is unreachable
+ * Added IB MLX5 as a separate UCX modules with RPM
+ * Added initial GGA support
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
+ * Implemented user-mode registration approach for XGVMI
+#### CUDA
+ * Improved GDR latency estimations on AMD Milan systems
+ * Improved reliability by detecting runtime/build GDR library version match
+ * Added handling missing IPC capability when unpacking keys
+ * Added caching for CUDA IPC memory pool import operation
+ * Added GDR environment variables to optimize performance on Grace Hopper systems
+ * Added CUDA Fabric memory support
+ * Added multi-node NVLINK support
+ * Improved peer accessible cache for CUDA IPC to be reentrant
+ * Added detection and allocation of Fabric VMM memory
+#### UCS
+ * Improve non-temporal data transfer implementation
+ * Allowed configuration with wildcard prefix and name matching
+ * Defined a non-inline function to allow ASAN validation
+ * Reduced stack size in topology detection code
+ * Improved bitmaps configuration parsing with wider bitfield
+ * Added options to set topology distance between devices
+ * Improve VFS unix socket watch by using user private folder
+ * Added general subnet matching support infrastructure
+ * Allowed user-based existing data copy when appending to array
+ * Improved time units description
+ * Added x86 non-temporal buffer transfers to skip cache
+#### UCM
+ * Improved log on memory remap failure
+ * Added memory hooks to track allocations by CUDA memory mapping functions
+#### Tools
+ * Improved performance by increasing window size for put_bw and add get_bw
+ * Added multi-send flag for receive operations in bandwidth benchmarks in ucx_perftest
+ * Improved ucx_perftest uni-directional test with added fence
+ * Improved ucx_perftest batch helper notes
+#### Documentation
+ * Added a section regarding adaptive routing on RoCE
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+#### AZP
+ * Improved repository setup when running CI
+ * Improved test_fuzzy reporting
+ * Updated to CUDA version 12.5.1 for Fabric memory tests
+ * Improved active IB HCA detection on CI
+ * Added DRP release pipeline
+ * Removed io_demo tests for ConnectX-4
+ * Added JUCX tests on ARM
+ * Improved verification and release processes
+ * Added wire-compat testing with v1.17.x
+ * Improved reports with added server-side log prints
```

Comment:
we do not mention AZP improvements in NEWs

Reply:
removed all AZP section

Reply:
remove AZP bug section